multiqc/modules/rna_seqc/rna_seqc.py

```diff
@@ -222,5 +252,8 @@ def plot_correlation_heatmap(self):
             self.add_section (
                 name = '{} Correlation'.format(corr_type),
                 anchor = 'rseqc-rna_seqc_correlation',
+                helptext = 'Spearman Correlation Coefficient of the data to the reference expression profile provided. '
+                           '(Will only be calculated and displayed on this page if a correlation comparison file is '
```

Comment:
I removed the helptext, I don't know what to say except this is a correlation plot and this doesn't help ^^